spec/spec_helper.rb

```diff
@@ -5,6 +5,11 @@
 
 require 'dotiw'
 
+if defined?(ActionView)
```

Comment:
Feels unnecessary here, it's already brought in by Rails via gemfiles/... .

Reply:
Move to _spec.rb.

```ruby
describe 'A better distance_of_time_in_words' do
  if defined?(ActionView)
    include ActionView::Helpers::DateHelper
    include ActionView::Helpers::TextHelper
    include ActionView::Helpers::NumberHelper

    require 'action_controller'
  else
    include DOTIW::Methods
  end
```

Reply:
Moved: https://github.com/radar/distance_of_time_in_words/pull/111/commits/8156131a3195c3be7fee271ecdd0cba853f62847